src/app/my-messages/member-kicked-out-notification.ts

```diff
@@ -0,0 +1,10 @@
+import { AbstractNotification } from './abstract-notification';
+import { Community } from '../communities/community';
+
+export class MemberKickedOutNotification extends AbstractNotification {
+  community: Community;
+
+  getTypeText(): string {
+    return 'Being licked-out of community';
```

Comment:
There is a misprint. Besides I suggest to have here "The user was kicked out of the community" message.

Reply:
LOL 😆 
thank you. done